als-server/shared/src/main/scala/org/mulesoft/als/server/modules/diagnostic/custom/CustomValidationManager.scala

```diff
@@ -40,6 +40,7 @@ class CustomValidationManager(override protected val telemetryProvider: Telemetr
                                      references: Map[String, DiagnosticsBundle],
                                      uuid: String): Future[Unit] = {
     val startTime = System.currentTimeMillis()
+    logger.debug(s"Starting custom validation for $uri", "CustomValidationManager", "3")
```

Comment:
should this be a telemetered message?
what is `"3"`?

Reply:
Fail xD

Reply:
It already is a telemetered message, so im removing this line.